homeassistant/components/sensor/gtfs.py

```diff
@@ -219,11 +223,16 @@ def icon(self):
         """Icon to use in the frontend, if any."""
         return ICON
 
+    @property
```

Comment:
Use `device_state_attributes()` to collect and present attributes in the frontend.